packages/payment-processor/src/payment/utils.ts

```diff
@@ -214,6 +216,30 @@ export function validateErc20FeeProxyRequest(
   }
 }
 
+/**
+ * Validates the amount and fee parameters for an Eth Fee Proxy based request.
+ * @param request to validate
+ * @param feeAmountOverride optionally, the custom fee amount
+ * @param paymentNetwork defaults to ETH Fee Proxy contract
+ */
+export function validateEthFeeProxyRequest(
```

Comment:
I feel this method should be in the eth-fee-proxy file

Reply:
I made it the same way as for erc20. Again a Refacto mission is required here :face_in_clouds:

Reply:
That is a good question. I would be better for this file but it makes the general organization even more messy.

Reply:
I will go for your point of view anyway :)

Reply:
you can put in a new file if you prefer. but the utils should not have specificities of PNs

Reply:
I agree, the questions was should I do it now or in a specific PR refactoring everything.

Reply:
I move it to this file as you suggest